java/system-test/system-test-dsl/src/test/java/sleeper/systemtest/dsl/sourcedata/RecordNumbersTest.java

```diff
@@ -14,22 +14,21 @@
  * limitations under the License.
  */
 
-package sleeper.systemtest.datageneration;
+package sleeper.systemtest.dsl.sourcedata;
 
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import java.util.stream.LongStream;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class RecordNumbersTest {
     @Test
     void shouldShuffleRecordsWithFixedRandomSeed() {
         // Given
         LongStream longStream = LongStream.of(1L, 2L, 3L, 4L, 5L);
 
         // When/Then
-        assertThat(RecordNumbers.scrambleNumberedRecords(longStream).stream())
+        Assertions.assertThat(RecordNumbers.scrambleNumberedRecords(longStream).stream())
```

Comment:
Can't this be statically imported like we usually do?

Reply:
Done, thanks, not sure how that happened.